icpc-wf/ansible/admin.yml

```diff
@@ -94,13 +99,7 @@
         owner: domjudge
         group: domjudge
         mode: 0644
-      register: css_installed
-
-    - name: Clear application cache
-      command: "{{DJ_DIR}}/webapp/bin/console cache:clear"
-      become: yes
-      become_user: domjudge
-      when: css_installed.changed
+      notify: clear application application
```

Comment:
This and below also have to be changed.

Reply:
It still says `clear application application` here.

Reply:
I think I need more sleep

Reply:
I just forgot to commit the file 🤣